static/templates/user_presence_row.handlebars

```diff
@@ -3,7 +3,7 @@
     <span class="user-status-indicator"></span>
     <span class="user-device-indicator">{{#if mobile}}{{/if}}</span>
       <a href="#" data-email="{{email}}" data-name="{{name}}" title="{{name}} {{type_desc}}"
-         class="{{#if my_fullname}} my_fullname{{/if}}">{{name}}</a>
+        class="{{#if my_fullname}} my_fullname{{/if}}">{{name}}{{#if starred}}(<span class="icon-vector-star"></span>){{/if}}</a>
```

Comment:
Yeah, I think the "pinned users" list style UI makes a lot of sense (and we just decided to go with something like that on the stream starring UI...).